### DISCUSSION TITLE
An error occurred while parsing the WebWorker bundle. This is most likely due to improper transpilation by Babel;

### DISCUSSION THREAD

**Discussion by marcel0799** [uber/react-map-gl#1550] [Q&A]
Hello, 

i am just getting started with react and as a project i wanted to create a heatmap that is displaying some of my data in 3d. 
To do that i want to use deck.gl on top of mapbox maps. Deck.gl is using the 'staticMap' from 'react-map-gl' as a basemap. 
Whenever i try to display any map from 'react-map-gl' (even with the provided simple examples from 'react-map-gl') i am getting the following erros. 

I have tried every suggestion from the Mapxbox doku: "https://docs.mapbox.com/mapbox-gl-js/guides/migrate-to-v2/"

Does anyone have any different Advice for me ?

![BugBabelMap](https://user-images.githubusercontent.com/44548916/129377620-cb38418e-823c-45aa-93ef-1ee98996b473.png)

**Comment by rachel-zhang-dev** [comment]
try this
https://docs.mapbox.com/mapbox-gl-js/guides/install/#excluding-mapbox-gl-js-explicitly-from-transpilation
